rs/src/gstd/mod.rs

```diff
@@ -41,3 +41,31 @@ impl ExecContext for GStdExecContext {
         *self.msg_id.get_or_init(msg::id)
     }
 }
+
+pub struct CommandResult<T>(T, ValueUnit);
+
+impl<T> CommandResult<T> {
+    pub fn new(value: T) -> Self {
+        Self(value, 0)
+    }
+
+    pub fn with_value(self, value: ValueUnit) -> Self {
+        Self(self.0, value)
+    }
+
+    pub fn to_tuple(self) -> (T, ValueUnit) {
+        (self.0, self.1)
+    }
```

Comment:
Don't like this. I'd prefer seeing something like
```
pub fn result() -> &T {...}
pub fn value() -> ValueUnit {...}
```

Reply:
The type is only used within a macro, whe we need some unused methods?